src/utils.tsx

```diff
@@ -69,6 +69,27 @@ const toPercentage = (val: number) => Math.round(val * 100) + '%'
 
 const toPrettyNumber = (val: number) => Math.round(val * 10) / 10
 
+const eventTypeNames = new Map<number, string>([
+  [0, 'Regional'],
```

Comment:
Hm, can't tell why these are wrong. Not sure if it is TBA's fault or ours.

Michigan subdivisions should be district championship division, but they are championship finals.
Michigan should be district championship, but it is district championship division.
Pacific Northwest should be district championship, but it is district championship division.